packages/material-ui/src/Box/Box.spec.tsx

```diff
@@ -1,6 +1,14 @@
 import * as React from 'react';
 import Box from '@material-ui/core/Box';
 
+interface TestProps {
+  test?: string;
+}
+
+const Test: React.FC<TestProps> = (props) => {
```

Comment:
Please use function declaration for components. [`React.FC` is problematic](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/).

Reply:
Done